ttnn/cpp/ttnn/operations/ccl/ccl_op_fusion.hpp

```diff
@@ -23,6 +23,7 @@ struct AllGatherFusedOpSignaler {
     uint32_t num_fused_op_cores_to_signal;
     std::vector<CoreCoord> fused_op_receiver_cores_noc;
     std::vector<uint32_t> fused_op_receiver_signal_semaphores;
+    bool mcast_fused_op_cores;
```

Comment:
missing initialization?

Reply:
there is a default initialization [here](https://github.com/tenstorrent/tt-metal/pull/12292/files#diff-f543504d02032becb23471a678b022b5e9365cc51e34bbfc155c32cbb6bfaf8cR40), but I guess it wouldn't hurt to also include it at the top level.